Website/Components/Post/Post.razor.css

```
@@ -1,24 +1,31 @@
 .card {
-  /* border: 1px solid; */
-  border-left: 0.5rem solid;
+  border-left: 0.4rem solid;
   border-color: grey;
   padding: 0.5rem;
   border-radius: 0;
+  background-color: #1b2042;
+}
+.card:hover {
+  background-color: red;
```

```suggestion
  background-color: #2b4080;
```
red hurts my eyeballs!

---

well just color you can change it to whatever.